### PR TITLE
Adding a new section to the markdown file

### PR DESCRIPTION
**Pull request by ghost**
I added a new section to the `file1.md` file.  I thought it would be useful to you.  Let me know what you think!

### CODE REVIEW

File: file1.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # This is a markdown file!
 
 This file has some contents.
+
+## This is a section!
```
Comment (kmpaul) [inline file1.md]: I don't really like this section title.  Could you change it to:

```suggestion
## This is a better section title
```

Comment (josephgrim) [inline file1.md]: I don't really like this section title.  Could you change it to:

```suggestion
## This is a better section title
```

File: file1.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+And it has contents, too!
```
Comment (kmpaul) [inline file1.md:7]: This content looks great!

**Review by kmpaul — CHANGES_REQUESTED** [review]
Thanks, @xdev-user!  This looks pretty good.  I've made some suggestions for changes.  Please get back to me when you can.

**Review by josephgrim — CHANGES_REQUESTED** [review]
Thanks, @josephgrim!  This looks pretty good.  I've made some suggestions for changes.  Please get back to me when you can.

File: file1.md (hunk 2 above)
Comment (ghost) [reply file1.md:7]: Thanks!

**Review by kmpaul — APPROVED** [review]
Thanks for the contribution, @xdev-user!  This looks great!

**Pull request merged**